content/article/zero-downtime-migrations.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Zero Downtime Migrations"
+date: "2018-10-30"
+type: "article"
+---
+
+## Introduction
+
+Say you are maintaining an e-commerce application that has a catalog of 100 products stored in a table named `product`
+in an RDBMS database, say PostgreSQL. Now, let's add a new column named `discount` to the table.
+Let's use this query:
+
+`ALTER TABLE product ADD COLUMN "discount" INTEGER NOT NULL DEFAULT 0;`
```

Comment:
For syntax highlighting of this SQL query, add the code block in this way instead:

```
    ```sql
     ALTER TABLE product ADD COLUMN "discount" INTEGER NOT NULL DEFAULT 0;
     ```
```

Please do this for all other code blocks.

Reply:
Done.